### PR TITLE
Add conditional import for aws lambda package

### PR DESCRIPTION
**Pull request by MCarlomagno**
Still requires some refactor/cleanup, I succesfully tested this logic running the following action and switching runtime in AWS console

```js
const { version } = require('node:process');

exports.handler = async function(credentials, context) {
  const creds = JSON.parse(credentials.credentials);
  
  console.log("node version", version);

  let lambda;
  let result;
  if (version >= 'v20.0.0') {
    const { Lambda } = require('@aws-sdk/client-lambda');
      lambda = new Lambda({
        credentials: {
          accessKeyId: creds.AccessKeyId,
          secretAccessKey: creds.SecretAccessKey,
          sessionToken: creds.SessionToken,
        },
      });
    
      result = await lambda.invoke({
          FunctionName: '< relayer proxy arn >',
          Payload: JSON.stringify({ action: 'get-self-status' }),
          InvocationType: 'RequestResponse',
        });
    
     result.Payload = result.Payload.transformToString();
    } else {
     const Lambda = require('aws-sdk/clients/lambda');
      lambda = new Lambda(
        creds
          ? {
            credentials: {
              accessKeyId: creds.AccessKeyId,
              secretAccessKey: creds.SecretAccessKey,
              sessionToken: creds.SessionToken,
            },
          }
          : undefined,
      );
      result = lambda.invoke({
        FunctionName: '< relayer proxy arn >',
        Payload: JSON.stringify({ action: 'get-self-status' }),
        InvocationType: 'RequestResponse',
      }).promise();
  }
  

  console.log("function errror", result.FunctionError)
  return result;
}
```

### CODE REVIEW

**Review by zeljkoX — COMMENTED** [review]
LGTM

Added few comments!

Just a note that we would need this for [defender-client https://github.com/OpenZeppelin/defender-client as well.

File: packages/base/src/action/index.ts [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+};
+
+type LambdaV3 = {
+  invoke: (params: { FunctionName: string; Payload: string; InvocationType: string }) => Promise<InvokeResponse>;
+};
+
+type LambdaLike = LambdaV2 | LambdaV3;
+
+function isLambdaV3Compatible() {
+  return version >= NODE_MIN_VERSION_FOR_V3;
```
[the remark this answers is not in the document]
Comment (MCarlomagno) [reply packages/base/src/action/index.ts:33]: I would do it processing the string, I'm sure we will forget this by node v30 😅

**Review by shahnami — APPROVED** [review]
LGTM!

**Review by zeljkoX — APPROVED** [review]
LGTM

Thanks for tackling this!

**Pull request merged**